core/src/HIP/Kokkos_HIP_Space.cpp

```diff
@@ -187,12 +187,13 @@ Kokkos::HIP::allocation WARNING: The combination of device and system configurat
         }
 
         // check for correct runtime environment
-        if (!Kokkos::Impl::xnack_enabled())
+        if (!Kokkos::Impl::xnack_environment_enabled())
           std::cerr << R"warning(
 Kokkos::HIP::runtime WARNING: Kokkos was not able to verify that xnack is enabled.
                               Without xnack enabled, Kokkos::HIPManaged might not behave as expected.
-                              Set HSA_XNACK=1 in your environment and ensure "CONFIG_HMM_MIRROR=y"
-                              is in the /boot/config file and that file is readable.)warning"
+                              Set HSA_XNACK=1 in your environment. For further information on HMM support
+                              call `Kokkos::print_configuration`, or run with KOKKOS_PRINT_CONFIGURATION=1
+                              in your environment.)warning"
```

Comment:
same here

Reply:
done in https://github.com/kokkos/kokkos/pull/7765/commits/adc02dd6d769fd477e54fd9bc8896b607e66b86b